ext/ldap/ldap.c

```diff
@@ -925,6 +925,10 @@ PHP_FUNCTION(ldap_connect)
 	ldap_linkdata *ld;
 	LDAP *ldap = NULL;
 
+	if (ZEND_NUM_ARGS() == 2) {
+	    zend_error(E_DEPRECATED, "Usage of ldap_connect with two arguments is deprecated.");
```

Comment:
```suggestion
	    zend_error(E_DEPRECATED, "Usage of ldap_connect with two arguments is deprecated");
```
Otherwise the deprecation message reads incorrectly.